force-app/main/default/classes/RD2_ETableController_TEST.cls

```diff
@@ -111,7 +114,7 @@ public class RD2_ETableController_TEST {
         RD2_ETableController.selector = (RD2_ERecurringDonationsSelector) Test.createStub(RD2_ERecurringDonationsSelector.class, mock);
        
         Test.startTest();
-        List<RD2_ETableController.TableView> result = RD2_ETableController.retrieveTableView();
+        List<RD2_ETableController.TableView> result = RD2_ETableController.retrieveTableView(RD2_ETableController_TEST.ALL_RD);
         Test.stopTest();
 
         System.assertNotEquals(null, result, 'Recurring Donations should not be null');
```

Comment:
Can we assert that the values in the TableView object are set correctly? The process of building this object calls methods that run logic to change the set values, we would benefit greatly from asserts to confirm that `frequency`,  `status` and `paymentMethod` values are set correctly.

Reply:
Yes @voduyemi I double checked it, and I also created one new scratch org with this code implementations to check the complete flow, and check this values too.